core/src/banking_stage.rs

```diff
@@ -540,15 +603,15 @@ impl BankingStage {
                                 working_bank,
                             ));
                         }
-                        new_tx_count += processed;
+                        new_tx_count += committed_transactions_count;
```

Comment:
Looks like this `new_tx_count` counter is primarily used here:
```
        banking_stage_stats
            .consumed_buffered_packets_count
            .fetch_add(new_tx_count, Ordering::Relaxed);
```
My understanding is that it's supposed represent how many packets are removed from the buffer (and not rebuffered). So I think we should be trying to include both of these two types of transactions:
```
    // 1) Did not execute due to some fatal error like too old, or duplicate signature.
    // 2) Were executed and committed, captured by `committed_transactions_count` below.
```

Reply:
> My understanding is that it's supposed represent how many packets are removed from the buffer (and not rebuffered).
Yeah, then maybe `transactions_attempted_execution_count` - `retryable_transaction_indexes` is a good measure of this, since the difference there is never rebuffered

I think then this should be moved and replace this metric here: https://github.com/solana-labs/solana/blob/f5169742cedb191790646d58f93ab95686b68fc1/core/src/banking_stage.rs#L1484-L1488